src/main.rs

```diff
@@ -187,8 +184,6 @@ fn process<P, I>(old_args: I, dep_path: P, sysroot: &str) -> Result<(), i32>
     args.push(String::from("--sysroot"));
     args.push(sysroot.to_owned());
     args.push("-Zno-trans".to_owned());
-    args.push("--cfg".to_owned());
-    args.push(r#"feature="clippy""#.to_owned());
```

Comment:
The problem also happens when something like `#![cfg_attr(feature="clippy", plugin(clippy))]` is in not in a dependency but directly in the crate.